### ISSUE TITLE
Issue with M3U8 contain POD1/POD2 Multiple map in one playlist

### ISSUE THREAD

**Issue by Azzuray** [nilaoda/N_m3u8DL-CLI#322]
Hi, first of all, great job for your tool ! Amazing work, but ive rand into an issue lately, i'm trying to download a playlist, which as multiple map inside and i think your tool doesnt handle well 

`#EXTM3U
#EXT-X-VERSION:6
#EXT-X-TARGETDURATION:9
#EXT-X-PLAYLIST-TYPE:VOD
#EXT-X-MEDIA-SEQUENCE:0
#EXT-X-PROGRAM-DATE-TIME:2019-01-01T00:00:00.000Z
#EXT-X-DATERANGE:ID="pod_0",START-DATE="2019-01-01T00:00:00.000Z",DURATION=3.003,X-START-OFFSET="0.0"
#EXT-X-KEY:METHOD=SAMPLE-AES-CTR,KEYFORMAT="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed",KEYFORMATVERSIONS="1",URI="data:text/plain;base64,AAAAMnBzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAABISENc7OanL30hQgeRTwz/Qmhk="
**#EXT-X-MAP:URI="70a48659-6d50-4228-8e5c-1f43abc32d47/83c9-BUMPER/02/4250K/map.mp4"**
#EXTINF:3.003,
70a48659-6d50-4228-8e5c-1f43abc32d47/83c9-BUMPER/02/4250K/00/00/00_000.mp4
#EXT-X-DISCONTINUITY
**#EXT-X-DATERANGE:ID="pod_1",START-DATE="2019-01-01T00:00:03.003Z",DURATION=6581.700125,X-START-OFFSET="3.003"
#EXT-X-MAP:URI="d585dde2-d29d-4501-a84b-40c5eb3aba5c/7a46-MAIN/02/4250K/map.mp4"**
#EXTINF:8.008,
d585dde2-d29d-4501-a84b-40c5eb3aba5c/7a46-MAIN/02/4250K/00/00/00_000.mp4`

**Comment by nilaoda** [comment]
Yes, I only identified the first `#EXT-X-MAP` and drop others. You can try manually editing M3u8 and use `--baseUrl` to download it.
BTW, N_m3u8DL-CLI can not decrypt `SAMPLE-AES-CTR`.

**Comment by Azzuray** [comment]
Yes, it's ok i know, i use your tool to skip this "encryption" only to get the files beacause ffmpeg dont support it (or never will), but the map/pod=id is realy annoying :/
